http-kernel-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,20 @@
+<?php
+    $resp = new Symfony\Component\HttpFoundation\Response();
+    $cookiepath = dirname($_SERVER['REQUEST_URI']) . '/';
```

Comment:
1. rename `$cookiepath` into `$cookiePath`
2. replace `$_SERVER['REQUEST_URI']` with `$request->server->get('REQUEST_URI')`

---

1. not according to a coding standard
2. incorrectly fixed from previous code review